packages/ui-app/src/styles/partials/_variables.css

```diff
@@ -24,3 +28,6 @@ $button-negative-color-hover: $middle-purple;
 $anchor-color: $light-slate-gray;
 $anchor-color-hover: $cadet-grey;
 $progress-bar-color: $dark-sky-blue;
+
+/* fallback font families included */
+$hash-font-family: monospace, 'Courier New', Courier;
```

Comment:
Actually `monospace` is whatever the browser decides it to be, so actually no need for a fallback, it is the built-in that is always there. https://developer.mozilla.org/en-US/docs/Web/CSS/font-family

So all these mixin changes can be reverted. Just `monospace` is fine and works by default.